### PR TITLE
feat: introduce the `--ignore-snapshots` option

### PR DESCRIPTION
**Pull request by aslushnikov**
This patch introduces `--ignore-snapshots` Playwright Test CLI option,
as long as `ignoreSnapshots` configuration argument.

### CODE REVIEW

File: packages/playwright-test/src/matchers/toMatchSnapshot.ts [hunk 1]
```diff
@@ -292,6 +296,10 @@ export async function toHaveScreenshot(
   const testInfo = currentTestInfo();
   if (!testInfo)
     throw new Error(`toHaveScreenshot() must be called during the test`);
+
+  if (testInfo.config._ignoreSnapshots)
+    return { pass: !this.isNot, message: () => '' };
```
Comment (dgozman) [inline packages/playwright-test/src/matchers/toMatchSnapshot.ts:301]: I wonder if we could wait for a screenshot or two consecutive matching screenshots to better match real timings.

File: tests/playwright-test/to-have-screenshot.spec.ts [hunk 2]
```diff
@@ -555,6 +555,20 @@ test('should fail on same snapshots with negate matcher', async ({ runInlineTest
   expect(result.output).toContain('Expected result should be different from the actual one.');
 });
 
+test('should not fail if --ignore-snapshots is passed', async ({ runInlineTest }) => {
+  const result = await runInlineTest({
+    ...playwrightConfig({ screenshotsDir: '__screenshots__' }),
+    '__screenshots__/a.spec.js/snapshot.png': redImage,
+    'a.spec.js': `
+      pwt.test('is a test', async ({ page }) => {
+        await expect(page).not.toHaveScreenshot('snapshot.png', { timeout: 2000 });
```
Comment (dgozman) [inline tests/playwright-test/to-have-screenshot.spec.ts:564]: I think this does not test anything, since screenshot does not match the red image anyway.

**Review by dgozman — APPROVED** [review]

File: packages/playwright-test/src/matchers/toMatchSnapshot.ts (hunk 1 above)
Comment (aslushnikov) [reply packages/playwright-test/src/matchers/toMatchSnapshot.ts:301]: I'd refrain from this for now.

File: tests/playwright-test/config.spec.ts [hunk 3]
```diff
@@ -373,6 +373,25 @@ test('should inerhit use options in projects', async ({ runInlineTest }) => {
...
+  const result = await runInlineTest({
+    'playwright.config.ts': `
+      module.exports = {
+        ignoreSnapshots: true,
+      };
+    `,
+    'a.test.ts': `
+      const { test } = pwt;
+      test('pass', async ({}, testInfo) => {
+        expect('foo').toMatchSnapshot();
```
Comment (mxschmitt) [inline tests/playwright-test/config.spec.ts:386]:
```suggestion
        expect('foo').toMatchSnapshot();
        expect('foo').not.toMatchSnapshot();
```

**Pull request merged**